### PR TITLE
Quiz question

### PR DESCRIPTION
**Pull request by maurizi**
## Overview

Adds a component to show quiz options and manage question state.

### Demo

Initial state:
mike-desktop_3474_

One wrong question:
mike-desktop_3474_ (2)

No wrong questions, hint shown:
mike-desktop_3474_ (1)

## Testing Instructions

 * Use the quiz tab of the site on Netlify

### CODE REVIEW

File: src/app/src/components/QuizQuestion.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            usedHint: false,
+        };
+    }
+
+    render() {
+        const { answer, choices, onDone } = this.props;
+        const { guessed, usedHint } = this.state;
+
+        const showHint = guessed.length > 0 || usedHint;
+        // TODO: Remove the default value here when we get hint text
```
Comment (maurizi) [inline src/app/src/components/QuizQuestion.js:60]: @fungjj92 it doesn't look like we have an issue already for the hint text, should I open one?

Comment (fungjj92) [reply src/app/src/components/QuizQuestion.js:60]: Yeah! Put a blocked tag please.

File: src/app/src/components/QuizQuestion.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            hint = 'Almost! ' + hint;
+        }
+        const options = choices.map((fish, idx) => (
+            <StyledQuizOption
+                key={idx}
+                fish={fish}
+                showImage={showHint}
+                shadeImage={guessed.includes(idx)}
+                onClick={() => {
+                    if (guessed.includes(idx)) {
```
Comment (fungjj92) [inline src/app/src/components/QuizQuestion.js]: Can we move this if/else block into a named function?

File: src/app/src/components/QuizQuestion.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                        return;
```
Comment (fungjj92) [inline src/app/src/components/QuizQuestion.js]: Nice detail, not letting the user guess the same wrong answer.

File: src/app/src/components/QuizQuestion.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                    {hint}
+                </Button>
+            </StyledQuizQuestion>
+        );
+    }
+}
+
+QuizQuestion.propTypes = {
+    answer: QuizFish.isRequired,
+    choices: PropTypes.arrayOf(QuizFish.isRequired),
```
Comment (fungjj92) [inline src/app/src/components/QuizQuestion.js]: You can import `arrayOf` directly, no ?

**Review by fungjj92 — COMMENTED** [review]
Looks great and works well!!

**Comment by maurizi** [comment]
@fungjj92 ready for another look

**Review by fungjj92 — APPROVED** [review]
Great !

**Pull request merged**